clang/test/dpct/tm-complex.cu

```diff
@@ -533,6 +535,8 @@ void foo_test_4() {
 // CHECK-NEXT:    /*
 // CHECK-NEXT:    DPCT1049:{{[0-9]+}}: The work-group size passed to the SYCL kernel may exceed the limit. To get the device limit, query info::device::max_work_group_size. Adjust the work-group size if needed.
 // CHECK-NEXT:    */
+// CHECK-NEXT:        if (!dpct::get_default_queue().get_device().has(sycl::aspect::fp64))
```

Comment:
how about introduce a util function  "capability_check(device, prop, ...);" to replace  line L538 and 539.

Reply:
fixed